CODINGSTYLE.md

```diff
@@ -0,0 +1,335 @@
+# Coding Style Guide
+
+The Scala.js project has pretty drastic rules in terms of coding style.
```

Comment:
Well, if you say 'highly encouraged' you're pretty much saying 'are forced to respect', except you sound kinder, but if you wanna be 100% transparent, then I think it would be okay to say something like 'strict policy' as long as you give a valid reason why.
something along the lines of (feel free to change):

.... has a strict policy regarding coding style, which is why it's still very maintainable even though it numbers $n LOCs ...